src/rancilio-pid.cpp

```diff
@@ -1920,6 +1932,11 @@ void setup() {
         previousMillisPressure = currentTime;
     #endif
 
+            // Init Scale by BREWMODE 2 or SHOTTIMER 2
```

Comment:
```suggestion
    // Init Scale by BREWMODE 2 or SHOTTIMER 2
```

Reply:
comment is a bit redundant but also doesn't match the condition on the next line